spec/requests/api/v0/instances_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+RSpec.describe "Api::V0::Instances", type: :request do
+  describe "GET /api/instance" do
+    it "returns the correct attributes", :aggregate_failures do
+      get api_instance_path
+
+      expect(response.parsed_body["cover_image_url"]).to eq Settings::General.main_social_image
+      expect(response.parsed_body["description"]).to eq Settings::Community.community_description
+      expect(response.parsed_body["logo_image_url"]).to eq Settings::General.logo_png
+      expect(response.parsed_body["name"]).to eq Settings::Community.community_name
+      expect(response.parsed_body["tagline"]).to eq Settings::Community.tagline
+      expect(response.parsed_body["version"]).to match(/(stable|beta|edge)\.\d{8}.\d+/)
```

Comment:
@jgaskins Shouldn't the dot after `\d{8}` also be escaped like the one before? As in "an actual dot", not any character. In its current form this regex also matches e.g. `edge.12345678x1`.

Reply:
Ah, yep, nice catch! @atsmith813 can you touch that up real quick? Just needs a `\` before the last dot. It's not critical, but it's definitely more correct.

Reply:
Updated, thank you!